src/formio/components/checkbox.ts

```diff
@@ -0,0 +1,17 @@
+import {InputComponentSchema} from '..';
+
+type Validator = 'required';
+type TranslatableKeys = 'label' | 'description' | 'tooltip';
+
+export type CheckboxInputSchema = InputComponentSchema<boolean, Validator, TranslatableKeys>;
+
+/**
+ * @group Form.io components
+ * @category Concrete types
+ */
+export interface CheckboxComponentSchema
+  extends Omit<CheckboxInputSchema, 'hideLabel' | 'disabled'> {
+  type: 'checkbox';
+  defaultValue?: boolean;
+  multiple?: false;
```

Comment:
do we need this key? I think I'd just not allow it at all.

Reply:
How can we know if it will not be there? I the admin frontend I see that the full schema still have this key

Reply:
with the existing formio builder it will (probably) be there, but we should not rely on it. With the new formio-builder, we control what gets set and what doesn't!

IMO the types should only describe what is relevant for our own builder/renderer in the context of Open Forms - they inform developers whether they need to consider certain behaviours or not. The `multiple` key being present in the type hints that the developer needs to account for `multiple` behaviour, but we don't actually allow that anywhere, so we risk doing too much.

Reply:
So then I should add it to the `Omit`? iirc it is still available in `FooInputSchema` (<- `StrictComponentSchema`<-`ComponentSchema`)

Reply:
it's already omitted from `StrictComponentSchema`, see https://github.com/open-formulieren/types/blob/main/src/formio/base.ts#L105 :-)